samples/Common.c

```diff
@@ -563,7 +563,9 @@ STATUS lookForSslCert(PSampleConfiguration* ppSampleConfiguration)
     STATUS retStatus = STATUS_SUCCESS;
     struct stat pathStat;
     UINT32 length = 0;
+#ifndef _WIN32
```

Comment:
We could use PIC primitives that are platform independent and can iterate over the directory. Not for this change though.

Reply:
For this one I am not sure what is the best way, we can use c++17 who provide filesystem API, or another way will be to use http://www.two-sdg.demon.co.uk/curbralan/code/dirent/dirent.c
who implement the functionality we need.
I think the second approach will be faster to do, @MushMal ?